.github/autoissuelabeler.yml

```diff
@@ -47,12 +47,36 @@
   title: "(?i).*mongo.*"
   description: "(?i).*mongo.*"
   notify: [loicmathieu]
-- labels: [area/openapi]
+- labels: [area/openapi, area/smallrye]
   title: "(?i).*openapi.*"
   notify: [EricWittmann,MikeEdgar,phillip-kruger]
-- labels: [area/graphql]
+- labels: [area/graphql, area/smallrye]
   title: "(?i).*graphql.*"
-  notify: [phillip-kruger,jmartisk]  
+  notify: [phillip-kruger,jmartisk]
+- labels: [area/reactive-messaging, area/smallrye]
+  title: "(?i).*reactive.messaging.*"
+  notify: [kenfinnigan,cescoffier,phillip-kruger]
+- labels: [area/tracing, area/smallrye]
+  title: "(?i).*(trace|tracing|telemetry).*"
+  notify: [kenfinnigan,Ladicek]
+- labels: [area/config, area/smallrye]
+  title: "(?i).*config.*"
```

Comment:
IMHO, this one is going to be a pain but you can try :).

Reply:
You mean it will match too much ?

Reply:
Yeah, that's what I'm worried about.

I even wonder if we don't have `config` in the default template. Oh yes, we do: https://github.com/quarkusio/quarkus/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.md&title= :)